orchestrator/core/database/load_seed_data.py

```
@@ -164,6 +164,39 @@ def load_seed_data(load_credentials: bool = True, load_platform_defaults: bool =
+                from database.database import get_db_session as _mk_session
+                from sqlalchemy import text as _sql
+                with _mk_session() as db:
+                    catalog_rows = db.execute(_sql("SELECT count(*) FROM marketplace_items")).scalar() or 0
+                if catalog_rows == 0:
+                    from scripts.seed_starter_agents import seed_starter_agents
+                    seed_starter_agents()
+                    print("  ✅ Starter agents seeded (empty catalog)")
+                from scripts.seed_marketplace_agents_v2 import seed_marketplace_agents_v2
+                seed_marketplace_agents_v2()
+                print("  ✅ Marketplace agents v2 seeded")
+            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)